Documentation/Routers/ReadWriteSplit.md

```diff
@@ -1058,7 +1058,7 @@ secondary servers. Readwritesplit is an exception to this rule. The following
 rules govern how readwritesplit behaves with servers that have different ranks.
 
 * Sessions will use the current primary server as long as possible. This means
-  that sessions with a secondary primary will not use the primary primary as long
+  that sessions with a secondary primary will not use the primary as long
```

Comment:
The duplication in this case, while horribly confusing, was intentional. You can replace the `primary primary` with `main primary`.